crypto/keys/mintkey/mintkey.go

```diff
@@ -116,27 +136,31 @@ func encryptPrivKey(privKey crypto.PrivKey, passphrase string) (saltBytes []byte
 }
 
 // Unarmor and decrypt the private key.
-func UnarmorDecryptPrivKey(armorStr string, passphrase string) (crypto.PrivKey, error) {
-	var privKey crypto.PrivKey
+func UnarmorDecryptPrivKey(armorStr string, passphrase string) (privKey crypto.PrivKey, algo string, err error) {
```

Comment:
undo named variables

Reply:
I feel without the named variable for algo, it might be unclear what that return value is

Reply:
please update the godoc comment with the return types as well